cpp/include/cudf/packed_types.hpp

```diff
@@ -0,0 +1,72 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2023-2025, NVIDIA CORPORATION.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <cudf/table/table.hpp>
+#include <cudf/types.hpp>
+#include <cudf/utilities/export.hpp>
+
+#include <rmm/device_buffer.hpp>
+
+#include <memory>
+#include <vector>
+
+namespace CUDF_EXPORT cudf {
+
+/**
+ * @addtogroup copy_split
+ * @{
+ * @file
+ * @brief Packed table and column types for serialization
+ */
+
+/**
+ * @brief Column data in a serialized format
+ *
+ * Contains data from an array of columns in two contiguous buffers: one on host, which contains
+ * table metadata and one on device which contains the table data.
```

Comment:
This is potentially used to store columns of different sizes, thus we should not assume it is used for "table".

Reply:
This was copied as-is from cpp/include/cudf/contiguous_split.hpp; I'd prefer not to modify it.